relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -264,8 +275,7 @@ pub(crate) fn extract_tags(span: &Span, config: &Config) -> BTreeMap<SpanTagKey,
         } else if span_op.starts_with("db") {
             span.description
                 .value()
-                .and_then(|query| sql_table_from_query(query))
-                .map(|t| t.to_lowercase())
```

Comment:
`to_lowercase` happens inside the function now, to prevent an additional string allocation.